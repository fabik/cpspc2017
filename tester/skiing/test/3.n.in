1
967507106 824003887 -983367357 -794600290
-556641720 152101765 147477275
