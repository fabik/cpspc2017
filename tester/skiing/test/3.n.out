3
967507106 824003887
152101765 147477275
-983367357 -794600290
